Jint/Native/Function/ScriptFunctionInstance.cs

```diff
@@ -58,6 +59,22 @@ protected internal override JsValue Call(JsValue thisArgument, JsValue[] argumen
             var strict = _functionDefinition.Strict || _thisMode == FunctionThisMode.Strict;
             using (new StrictModeScope(strict, true))
             {
+                // check the stack for the presence of a DelegateWrapper with SpreadParameters set to true
+                if (_engine.CallStack.TryPeek(out var stackItem) &&
```

Comment:
this does not seem the correct place to check for interop, JS spec and interop are generally different things and interop should prepare correct parameters for interop

Reply:
Yeah! I wondered that too... :) I changed it to Interop. Due to ScriptFunctionInstance is created early to bind a JS function, is not posible to achieve this kind of interop, because happens inside the call.